var/spack/repos/builtin/packages/httpie/package.py

```diff
@@ -13,6 +13,7 @@ class Httpie(PythonPackage):
     pypi = "httpie/httpie-2.6.0.tar.gz"
     maintainers("BoboTiG")
 
+    version("3.2.1", sha256="c9c0032ca3a8d62492b7231b2dd83d94becf3b71baf8a4bbcd9ed1038537e3ec")
     version("2.6.0", sha256="ef929317b239bbf0a5bb7159b4c5d2edbfc55f8a0bcf9cd24ce597daec2afca5")
     version("2.5.0", sha256="fe6a8bc50fb0635a84ebe1296a732e39357c3e1354541bf51a7057b4877e47f9")
     version(
```

Comment:
Need to update deps:
```python
install_requires = [                                                                     
    'pip',                                                                               
    'charset_normalizer>=2.0.0',                                                         
    'defusedxml>=0.6.0',                                                                 
    'requests[socks]>=2.22.0',                                                           
    'Pygments>=2.5.2',                                                                   
    'requests-toolbelt>=0.9.1',                                                          
    'multidict>=4.7.0',                                                                  
    'setuptools',                                                                        
    'importlib-metadata>=1.4.0; python_version < "3.8"',                                 
    'rich>=9.10.0'                                                                       
]                                                                                        
install_requires_win_only = [                                                            
    'colorama>=0.2.4',                                                                   
]  
```

Reply:
> 'importlib-metadata>=1.4.0; python_version < "3.8"', 

@adamjstewart how would you implement a `when="^python@:3.8"` for this dependency?

Reply:
This would look like:
```python
depends_on("py-importlib-metadata@1.4:", when="@X.Y: ^python@:3.7", type=("build", "run"))
```
where `X.Y` is the first version where this dependency was introduced. You can prob just use `@3:` if you don't know.

Reply:
Added!